blazor/datagrid/how-to/customize-filter-popup-using-css.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Customize filter popup using CSS in Blazor DataGrid | Syncfusion
+description: Learn here all about Customizing filter popup using CSS in Syncfusion Blazor DataGrid component and more.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Customize Filter Popup Using CSS
+
+You can set the height and width of each column in the filter popup dialog using the CSS style.
```

Comment:
explaination also not clear

Reply:
Can we remodify this line as below "You can set the height and width of a particular column’s filter dialog using the CSS style in the OnActionBegin event of Grid".  OnActionBegin will betriggered when certain actions gets initiated. THis event will be triggered before opening a filter dialg with RequestType as FilterBeforeOpen. At that timewe have enabled boolean variable to apply customized style to filter dialog.